### PR TITLE
Enhance documentation for Db2 on Cloud Field Mapping #821

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/DB2OnCloudSetup.md [hunk 1]
```diff
@@ -167,9 +167,45 @@ The FHIR server uses a proxy datasource mechanism, allowing new datasources to b
...
+```
+
+Use the following table to populate your datasource. 
+
+| IBM FHIR Server Configuration  | Description |
+|----|----|
+| serverName | from the credential object select `hostname` |
+| portNumber | from the credential object select  `port` |
+| databaseName | from the credential object select  `db`, generally always `BLUDB` |
+| apiKey | from the `password` field |
```
Comment (lmsurpre) [inline docs/src/pages/guides/DB2OnCloudSetup.md]: I don't think this is right.  Instead, it should be the API Key generated for the user created in the FHIRUSER group (i.e. the IBMid user created from https://github.com/IBM/FHIR/blob/master/docs/src/pages/guides/DB2OnCloudSetup.md#fhirserver-user-and-api-key )

File: docs/src/pages/guides/DB2OnCloudSetup.md [hunk 2]
```diff
@@ -98,28 +101,25 @@ You should now be able to connect to the database as the FHIRSERVER user using o
...
 ### Configuring a Liberty Datasource with API Key
 
-The FHIR export feature utilizes Java Batch (JSR-352) provided by the batch-1.0 feature in Liberty Profile. The JPA persistence layer can be configured to use Db2 as follows:
+The IBM FHIR Server Import/Export feature utilizes Java Batch (JSR-352) provided by the batch-1.0 feature in Liberty Profile. The persistence layer can be configured to use Db2 as follows:
 
 Create a Db2 user (e.g. FHIRBATCH) and associate it with a ServiceId (no need to create an Administration user, a simple user has sufficient privileges). Using a valid API-KEY for the given ServiceId, configure a new datasource and the Java Batch persistence layer as follows:
 
-```
-
+``` xml
```
Comment (lmsurpre) [inline docs/src/pages/guides/DB2OnCloudSetup.md]: maybe we should explain where this file is by default?

File: docs/src/pages/guides/DB2OnCloudSetup.md (hunk 1 above)
Comment (lmsurpre) [reply docs/src/pages/guides/DB2OnCloudSetup.md]: The new language looks good, thanks.

**Comment by prb112** [comment]
taking care of locally, and will push

**Pull request merged**